traitsui/testing/tester/wx/helpers.py

```diff
@@ -391,3 +391,30 @@ def key_click_slider(control, interaction, delay):
         wx.wxEVT_SCROLL_CHANGED, control.GetId(), position
     )
     wx.PostEvent(control, event)
+
+
+def readonly_displayed_text_handler(wrapper, interaction):
```

Comment:
The function does not need to know about ReadonlyEditor or query.DisplayedText or UIWrapper. This allows the functions to be more easily reused in other context. e.g. All you need to call this function is a wx.TextCtrl or wx.StaticText. If the widget is not placed on `wrapper.target.control`, but somewhere else, you can still reuse this function.

So I'd probably change the argument to just `(control)` (the wx widget), especially since the `interaction` isn't used at all. The function can be renamed such that it is more obvious about that the fact it is serving a read-only textbox. In the case of wx, one can do so via wx.TextCtrl or wx.StaticText. This function serves to normalize the lack of common interface between the two.